package.json

```diff
@@ -17,58 +19,60 @@
   "author": "Parity Technologies <info@ethcore.io>",
   "license": "GPL3",
   "devDependencies": {
-    "babel-core": "^6.22.1",
-    "babel-eslint": "^7.1.1",
-    "babel-loader": "^6.2.10",
-    "babel-plugin-transform-class-properties": "^6.22.0",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-plugin-transform-object-rest-spread": "^6.22.0",
-    "babel-plugin-transform-runtime": "^6.22.0",
-    "babel-polyfill": "^6.22.0",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-es2016": "^6.16.0",
-    "babel-preset-es2017": "^6.16.0",
-    "babel-preset-react": "^6.22.0",
-    "babel-preset-stage-0": "^6.16.0",
-    "babel-register": "^6.18.0",
-    "babel-runtime": "^6.22.0",
-    "case-sensitive-paths-webpack-plugin": "^1.1.4",
-    "chai": "^3.5.0",
+    "babel-core": "6.22.1",
+    "babel-eslint": "7.1.1",
+    "babel-loader": "6.2.10",
+    "babel-plugin-transform-class-properties": "6.22.0",
+    "babel-plugin-transform-decorators-legacy": "1.3.4",
+    "babel-plugin-transform-object-rest-spread": "6.22.0",
+    "babel-plugin-transform-runtime": "6.22.0",
+    "babel-polyfill": "6.22.0",
+    "babel-preset-es2015": "6.18.0",
+    "babel-preset-es2016": "6.16.0",
+    "babel-preset-es2017": "6.16.0",
+    "babel-preset-react": "6.22.0",
+    "babel-preset-stage-0": "6.16.0",
+    "babel-register": "6.18.0",
+    "babel-runtime": "6.22.0",
+    "case-sensitive-paths-webpack-plugin": "1.1.4",
+    "chai": "3.5.0",
     "chrome-extension-utils": "https://github.com/ngotchac/create-chrome-extension.git",
```

Comment:
Should we use the commit hash here ? (to stay in line with _locking_ the dependencies versions)